### ISSUE TITLE
Unable to contribute, with sphynx configuration error

### ISSUE THREAD

**Issue by macetw** [gcovr/gcovr#649]
**Describe the bug**
I am trying to contribute code, but the "make qa" step is not working, because sphynx is complaining that language is not defined. It is a warning escalated to an error.

**To Reproduce**
1. My environment is on MacOS, with python3 and gcc-8
2. I ran make dev-setup okay
3. I ran make qa:
```
~/gcovr$ make qa
cd doc && make html O=-W
make[1]: Entering directory '/home/macetw/gcovr/doc'
gcovr -h > examples/gcovr.out
Running Sphinx v5.1.1

Warning, treated as error:
Invalid configuration value found: 'language = None'. Update your configuration to a valid language code. Falling back to 'en' (English).
make[1]: *** [Makefile:27: html] Error 2
make[1]: Leaving directory '/home/macetw/gcovr/doc'
make: *** [Makefile:94: doc] Error 2
```

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - GCC version [e.g. MinGW 8.1.0]
 - GCOVR version [e.g. 4.2]
 - Project directory layout:
   - Roots
   - Object directories

**Additional context**

**Comment by macetw** [comment]
NOTE: I am not up to date with the 5.2 version code.  Trying to reproduce with the new steps. Probably can close.

**Comment by macetw** [comment]
Yep. These steps are totally different. Closing issue.

**Comment by Spacetown** [comment]
This was fixed with #639. Please rebase.

**Comment by macetw** [comment]
I really like this nox-based build. It's working very nicely.
